### ISSUE TITLE
Found REAL solution for cerificate error....

### ISSUE THREAD

**Issue by Dutchglory** [PixelTomsen/omeconomy-module#1]
after some heavy googeling i found it.  :) 
if you import certificates from virwox.com as normal user 
(as root is optional),  the money module works 

(on linux) 
just run: " certmgr -ssl https://www.virwox.com:419 "  
and add certificates.. 

see output here: http://db.tt/9mmUakju
maybe add this to readmefile

thank you

**Comment by PixelTomsen** [comment]
- opensim should never be run as root user
- create separate user for run this instance and try with this user mozroots

regards;-)

**Comment by Dutchglory** [comment]
true, i'm running as normal user...  you also need the second line for omeconomy-module to work...

**Comment by PixelTomsen** [comment]
I do not need ... (tested with reinstalling the linux on the new server) ... mozroots command enough
